### PR TITLE
Added tests for `@json` field testcase in framing

### PR DESCRIPTION
**Pull request by pasquale95**
As suggested in the issue #142, I've added a test collection to better define what is expected when framing over a `@json` field.

### CODE REVIEW

File: tests/frame/0069-frame.jsonld [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "@context": {
+    "ex": "http://example.org/vocab#"
```
Comment (gkellogg) [inline tests/frame/0069-frame.jsonld:3]: Need to include `"ex:info": {"@type": "@json"}` in the frame context for it to be used to compact the result.

Comment (gkellogg) [reply tests/frame/0069-frame.jsonld:3]: However, this results in an error in my implementation: " invalid frame: Invalid JSON-LD frame syntax; invalid value of `@type`: `@json`", which is likely either an implementation or a spec bug.

**Comment by gkellogg** [comment]
I updated your branch with the manifest entries required. Without the suggested update, the result will be in the expanded form:

```json
{
  "@context": {
    "ex": "http://example.org/vocab#"
  },
  "@id": "http://example.org/test/#library",
  "@type": "ex:Library",
  "ex:info": {
    "@value": {
      "author": "JOHN",
      "pages": 200
    },
    "@type": "@json"
  }
}
```

But, it fails with the update, which will require some investigation.

**Comment by gkellogg** [comment]
I've updated this to be a passing test.

**Review by gkellogg — APPROVED** [review]

**Review by dlongley — APPROVED** [review]

**Pull request merged**